core/runtime.rs

```diff
@@ -722,7 +692,13 @@ impl JsRuntime {
       JsRealm::new(v8::Global::new(scope, context))
     };
 
+    if let Some(m) = self.module_map.as_ref() {
+      m.borrow().loader.allow_ext_resolution();
+    }
```

Comment:
I believe these should be `.unwrap()` here - if module_map is not present during realm creation it's an internal bug.